packages/terra-application-docs/src/terra-dev-site/tool/terra-dev-site/UpgradeGuides.f/v7.0.0.s.tool.mdx

```diff
@@ -0,0 +1,148 @@
+|`appConfig.title`|`titleConfig.title`|Moved to a titleConfig object to align with terra-application-navigation config.|
+|`appConfig.favicon`|`faviconFilePath`|No longer nested under appConfig. The name now includes 'filePaths' to better describe the expected input.|
+|`appConfig.themes`|**removed**|The available themes are now pulled from the terra-theme.config file via the webpack global variable `TERRA_THEME_CONFIG`.|
+|`appConfig.defaultTheme`|`defaultTheme`|No longer nested under appConfig.|
+|`appConfig.locales`|**removed**|Depreicated as of terra-dev-site 6.23, now removed.|
+|`appConfig.defaultLocale`|`defaultLocale`|No longer nested under appConfig.|
+|`appConfig.defaultDirection`|`defaultDirection`|No longer nested under appConfig.|
+|`appConfig.extensions`|`extensionItems`|Renamed to align with terra-application-navigation config.|
+|`appConfig.extensions.iconPath`|`extensionItems.iconPath`|Renamed to align with terra-application-navigation config.|
+|`appConfig.extensions.key`|`extensionItems.key`|Renamed to align with terra-application-navigation config.|
+|`appConfig.extensions.text`|`extensionItems.text`|Renamed to align with terra-application-navigation config.|
+|`appConfig.extensions.componentPath`|`extensionItems.modalFilePath`| Rename to prepare for application v2 modals|
+|`appConfig.extensions.size`|**removed**|One size fits all|
+|`appConfig.headHtml`|`headHtml`|No longer nested under appConfig.|
+|`includeTestEvidence`|**removed**|Remove the ability to include test evidence, this should be covered now by a separate site report.|
+
+## Dependency changes
+
+Previously `terra-dev-site` was providing a version of `rect-intl` as a dependency. Because this satisfied the `terra-application` peer dependency requirement, many applications did not define it. With this release we have removed that dependency and applications will have to provide that peer dependency themselves.
+
+`terra-application` is now a peer dependency and must be defined by the package consuming terra-dev-site.
+
+`terra-toolkit` is no longer a peer dependency to allow consumers to use `terra-dev-site` v7 with either terra-toolkit or the replacement `terra-functional-testing` package.
+
+### Resulting Test changes
+
+The page introduces a new tab stop. Any tests using tabstops will have to be updated to include an additional tab stop.
+
+If you happen to be testing md or mdx components, a wrapping div has been removed and screenshots will have to be regenerated. I assume this is rare beyond `terra-dev-site` itself.
+
+## Node 10
+
+Node 10 is now the minium supported node version. This was bumped to support version 5 of react-docgen, which is used to generate props tables.
+
+## MDX
+
+The MDX loader is now used to load both `.md` and .`mdx` files. This may cause build errors in existing .md documents. This change allows us to remove the marked loader from the plugin.
```

Comment:
What build errors might they encounter in existing .md documents and how can they fix them?

Reply:
Good point, i have this documented [here](https://terra-applic-terra-dev--iotyfw.herokuapp.com/dev_tools/cerner-terra-application-docs/terra-dev-site/writing-documentation#mdx-loader) and i should link to it.